modules/shared/php/views/module-header.php

```diff
@@ -5,6 +5,8 @@
 ?>
 
 <div class="wrap vip-workflow-admin">
+	<?php /* ToDo: Convert this message into a proper notice for errors and success messages */ ?>
```

Comment:
Realized this was removed in my previous PR so I've added it back in

Reply:
FYI this file got blown away in https://github.com/Automattic/vip-workflow-plugin/pull/34, but we're [doing the same thing manually](https://github.com/Automattic/vip-workflow-plugin/blob/e99a1ef82de055e6207080fd0e587bbcec069aa1/modules/settings/views/settings.php#L20-L22) in the settings module.